ONSdigital/census-rm-sample-loader
# Send Reminders to SPG Estab Type Military SFA

# Motivation and Context
Need to be able to target Military SFA type SPGs with reminders

# What has changed
New estab type included in sample generator
New validator
Test for validator

# How to test?
Load a sample with "Military SFA" as ESTAB_TYPE and check file generates with no errors

# Links
https://trello.com/c/zxsbB9uO/985-reminders-send-reminders-to-spg-estabtype-military-sfa-5
https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?spaceKey=CRM&title=ESTAB_TYPES+2021
https://collaborate2.ons.gov.uk/confluence/display/SDC/RMC-293+-+Print+reminder+letters+and+questionnaires
